File: leetcode/1.两数之和.go
func twoSum(nums []int, target int) []int {
	hh := make(map[int]int)
	for i, v := range nums {
		if idx, ok := hh[target - v]; ok {
			return []int{idx, i}
		}
		hh[v] = i
	}
	return []int{}
}

File: leetcode/1.两数之和.py
#
# @lc app=leetcode.cn id=1 lang=python3
#
# [1] 两数之和
#

# @lc code=start
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dic = {}
        for i, x in enumerate(nums):
            if target - x in dic:
                return [i, dic[target-x]]
            dic[x] = i
        return []